build-logic/src/main/kotlin/kotlinx/io/conventions/kotlinx-io-publish.gradle.kts

```
@@ -92,8 +92,8 @@ fun RepositoryHandler.configureMavenPublication( project: Project) {
         }
     }
 
-    // Something that's easy to "clean" for development, not mavenLocal
-    maven("${project.rootProject.buildDir}/repo") {
+    // Something that's easy to "clean" for development not mavenLocal
+    maven("${project.layout.buildDirectory}/repo") {
```

```suggestion
    maven(project.rootProject.layout.buildDirectory.dir("repo")) {
```
So that it will be published to one place in `rootProject/build` (like it was before)